scripts/superdesk-authoring/metadata/metadata.js

```diff
@@ -630,6 +630,10 @@ function MetadataService(api, $q) {
             });
         },
         removeSubjectTerm: function(term) {
+            if (!this.subjectScope) {
```

Comment:
@sjunaid had to add this - in case there is no authoring opened there is an error